File: pset6/pset6/cash/cash.py
from cs50 import get_float, get_int
import math

cash = -1
while cash < 0:
    cash = get_float("Amount: ")
cash = cash * 100
cash = math.floor(cash)

quarters = cash // 25
dimes = (cash % 25) // 10
nickels = ((cash % 25) % 10) // 5
pennies = (((cash % 25) % 10) % 5) // 1
coins = quarters + dimes + nickels + pennies

print(coins)

File: pset1/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    // Ask for the amount of money
    float cash;
    do
    {
        cash = get_float("Amount: ");
    }
    while (cash < 0);
    //
    // Convert in pennies
    int pennies = round(cash * 100);
    printf("%i\n", pennies);
    //
    // Define change
    int quarter = 25;
    int dime = 10;
    int nickel = 5;
    int penny = 1;
    int coins = 0;
    //
    // Calculate coins
    while (pennies >= 25) 
    {
        pennies = pennies - quarter;
        coins++;
    }
    while (pennies >= 10) 
    {
        pennies = pennies - dime;
        coins++;
    }
    while (pennies >= 5) 
    {
        pennies = pennies - nickel;
        coins++;
    }
    while (pennies >= penny) 
    {
        pennies = pennies - penny;
        coins++;
    }

    printf("%i\n", coins);
}
